x-pack/plugin/ent-search/src/main/java/org/elasticsearch/xpack/entsearch/engine/action/PutEngineAction.java

```diff
@@ -133,6 +133,7 @@ public XContentBuilder toXContent(XContentBuilder builder, Params params) throws
             return builder;
         }
 
+        // TODO: How do I return RestStatus.CONFLICT here?
```

Comment:
DocWriteResponse does not currently include Conflict as a response. How else can we get this?